libs/model/src/utils/utils.ts

```diff
@@ -140,7 +140,7 @@ export async function getThreadContestManagers(
             WHERE ct.topic_id = :topic_id
             AND cm.community_id = :community_id
             AND cm.cancelled = false
-            AND cm.ended = false
+            AND cm.ended IS NULL;
```

Comment:
can ended be false? maybe coalesce(cm.ended, false) = false if that's the case

Reply:
coalesce actually didn't work but I changed it to check for either null or false